test/Assertions.php

```diff
@@ -26,6 +26,9 @@
 // phpcs:ignore WebimpressCodingStandard.NamingConventions.Trait.Suffix
 trait Assertions
 {
+    /**
+     * @psalm-return ObjectPropertyHydrator::class|ObjectProperty::class
```

Comment:
```suggestion
     * @psalm-return ObjectPropertyHydrator::class&ObjectProperty::class
```

(should be an intersection type)

Reply:
Never mind - this is detailing a class string name, so union makes sense.